mongo/integration/client_side_encryption_prose_test.go

```diff
@@ -1035,6 +1036,50 @@ func TestClientSideEncryptionProse(t *testing.T) {
 			})
 		}
 	})
+
+	// These tests only run when a KMS mock server is running on localhost:8000.
+	mt.RunOpts("kms tls tests", noClientOpts, func(mt *mtest.T) {
```

Comment:
Can we skip the entire test if `KMS_TLS_TESTCASE` is unset? It's minor, but we should generally try to skip as early as possible to make the test output easier to read.

Reply:
Good point! Added.